examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_site_security_privacy_policy/code/federated-policies/jobs/src/fl_job.py

```diff
@@ -18,7 +18,7 @@
 from src.fedavg import FedAvg
 from src.network import SimpleNetwork
 
-from nvflare.job_config.api import FedJob
+from nvflare.job_config import FedJob
```

Comment:
suggest change to from nvflare import FedJob, and suggest change this in another PR.